packages/assets-controllers/src/TokensController.test.ts

```diff
@@ -1657,36 +1703,32 @@ describe('TokensController', () => {
 
       mockContract([asset, anotherAsset]);
 
+      const registerListeners = new Promise<void>((resolve) => {
```

Comment:
`registerListeners` sounds like a function, but it's not. Perhaps `promiseForApprovals` is a better name?

Reply:
Fixed here: https://github.com/MetaMask/core/pull/3690/commits/1b7d9a50b35d19a9ec663f54cb1d76181980498b